docs/gallery/interactive/fast_fourier_transform.ipynb

```diff
@@ -0,0 +1,266 @@
+    "* `'rms'`:\n",
+    "        Normalize the spectrum to show the RMS value of the pure tone components contained in a signal. If the signal is a sine with an amplitude of 1, the spectrum will have an absolute value of $1/\\sqrt{2}$ (-3 dB) at the frequency of the sine.\n",
+    "* `'power'`:\n",
+    "        In a dB representation, it equals the `'rms'` normalization describing a power quantity. For stochastic broadband signals, this results in a dependence of the magnitude on the sampling rate as these are defined by a constant power density (see `'psd'`).\n",
+    "* `'psd'`:\n",
+    "        Using this normalization, signals are represented as *power densities* (e.g. in Pa²/Hz), leading to a meaningful representation for broadband stochastic signals independently of the sampling rate."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Overview: Appropriate FFT normalizations\n",
+    "\n",
+    "| Signal type            | Variation                              | Normalization          |\n",
+    "|------------------------|----------------------------------------|------------------------|\n",
+    "| Energy                 | Impulse responses / transfer functions | `'none'`               |\n",
+    "| Power                  | Discrete tones                         | `'amplitude'`, `'rms'`, `'power'` |\n",
+    "| Power                  | Broadband stochastic signals           | `'psd'`              |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examples\n",
+    "Four signals with a length of 100 samples and a sampling rate of 10 kHz are used for illustration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "\n",
+    "# set number of samples and sampling rate\n",
+    "n_samples = 1e3\n",
+    "sampling_rate = 10e3"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An **impulse** ($x(0)=1$ and zero otherwise) with a constant spectrum. This is an energy signal, so the appropriate normalization is `'none'` (the default)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "impulse = pf.signals.impulse(n_samples, sampling_rate=sampling_rate)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A **fractional octave FIR filter** presenting a system with finite energy (e.g., a loudspeaker transfer function, a room impulse response, an HRTF ...). It is an energy signal, so the appropriate normalization is `'none'`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filter = pf.dsp.filter.fractional_octave_bands(\n",
```

Comment:
```suggestion
    "filtered = pf.dsp.filter.fractional_octave_bands(\n",
```
filter should be renamed, since it's a build in function and could lead to confusions...